app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -2,11 +2,46 @@
 
 module Api
   module V1
-    class LobbiesController < AuthenticatedController
-      before_action :require_authentication, except: [:from_code, :join, :answer]
+    class LobbiesController < AuthenticatedController # rubocop:disable Metrics/ClassLength
+      before_action :require_authentication, except: [:players_done, :from_code, :join, :answer, :score]
       before_action :require_authorisation, only: [:show, :update, :destroy, :start]
       before_action :set_quiz, only: [:index, :create]
 
+      # rubocop:disable Metrics/AbcSize
+      # rubocop:disable Metrics/MethodLength
```

Comment:
You might be able to solve these by having `lobby` and `question` set in `before_action`s